concordium-consensus/src/Concordium/Scheduler.hs

```diff
@@ -2750,12 +2750,20 @@ handleChainUpdate (WithMetadata{wmdData = ui@UpdateInstruction{..}, ..}, mVerRes
                                 ValidatorScoreParametersUpdatePayload u -> case sIsSupported SPTValidatorScoreParameters scpv of
                                     STrue -> checkSigAndEnqueue $ UVValidatorScoreParameters u
                                     SFalse -> return $ TxInvalid NotSupportedAtCurrentProtocolVersion
+                                CreatePLTUpdatePayload payload ->
+                                    if protocolSupportsPLT (protocolVersion @(MPV m))
+                                        then do
+                                            -- TODO Check signature when relevant update keys are introduced.
+                                            -- Process update
+                                            createPLTUpdate payload
```

Comment:
Suggestion: I'm not sure that this should be a monolithic operation here. For instance, at least resolving the governance account address could be handled in the scheduler.

Reply:
I have added a bit more and some comments to how I expect the logic to develop, but it depends a lot on the other pieces especially the token module I guess.
Please have a look again